app/lib/common/widget/AppCommon.dart

```diff
@@ -1,13 +1,49 @@
+import 'dart:convert';
+import 'dart:math';
+
 import 'package:flutter/material.dart';
-import 'package:google_fonts/google_fonts.dart';
 
 Widget navBarTitle(String title) {
   return Text(
     title,
-    style: GoogleFonts.montserrat(
+    style: const TextStyle(
       fontSize: 20,
       fontWeight: FontWeight.w700,
       color: Colors.white,
     ),
   );
 }
+
+String getNameFromId(String name) {
+  int start = 0, end = 0;
+  if (name.contains('@')) {
+    start = name.indexOf('@') + 1;
+  }
+  if (name.contains(':')) {
+    end = name.indexOf(':');
+  }
+  return name.substring(start, end);
+}
+
+String formatedTime(int value) {
```

Comment:
I haven't checked this package but I initially used [duration](https://pub.dev/packages/duration) and it was not correctly parsing the seconds into hours (the calculated was 5 hours less than actual).  So I came up with own custom solution. I can open ticket for it if we want to use that instead of custom function.